srtcore/api.cpp

```diff
@@ -771,11 +771,11 @@ int CUDTUnited::newConnection(const SRTSOCKET listen, const sockaddr_any& peer,
 #if ENABLE_LOGGING
        static const char* why [] = {
            "UNKNOWN ERROR",
-           "CONNECTION REJECTED",
+           "INTERNAL REJECTION",
            "IPE when mapping a socket",
            "IPE when inserting a socket"
        };
-       LOGC(mglog.Error, log << CONID(ns->m_SocketID) << "newConnection: connection rejected due to: " << why[error]);
+       LOGC(mglog.Note, log << CONID(ns->m_SocketID) << "newConnection: connection rejected due to: " << why[error]);
```

Comment:
Maybe Warn?

Reply:
I agree in this case - this is a rejection response, so the listener did take an action.